josm.openstreetmap.de.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: josm.openstreetmap.de
+branch: stable
+tags:
+  - osm
+runtime: org.freedesktop.Platform
+runtime-version: 18.08
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk10
+command: josm
+rename-desktop-file: josm.desktop
+rename-appdata-file: josm.appdata.xml
+rename-icon: josm
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+
+build-options:
+    env:
+        JAVA_HOME: /usr/lib/sdk/openjdk10/
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk10/install.sh
+
+  - name: josm
+    disabled: true
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk10/enable.sh
+      # - ant/bin/ant dist
+
+    sources:
+      - type: git
+        url:  git://github.com/openstreetmap/josm.git
+      - type: archive
+        url:  https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.2-bin.tar.xz
+        sha256: 361c8ad2ed8341416e323e7c28af10a8297170a80fdffba294a5c2031527bb6c
+        dest: ant
+
+  - name: josm-binary
+    buildsystem: simple
+    build-commands:
+        - install -D josm.sh /app/bin/josm
+        - install -D josm-snapshot-14382.jar /app/bin/josm.jar
+        - install -D josm.desktop /app/share/applications/josm.desktop
```

Comment:
You could install the appdata file, the desktop file with the correct name and remove rename-* from the manfiest

Reply:
right. Do you think it's the better way of doing it? I have no feelings for either way.
Somehow it felt more natural to use the built-in rename-* facilities than messing around with the files myself. I somehow expect the rename-* stuff to do more than a mere renaming. And even if they do not right now, such behaviour could be added in the future.

Reply:
`rename-*` is a hack, all data files should be correct to start with.

Reply:
> You could install the appdata file, the desktop file with the correct name and remove rename-* from the manfiest

done.